src/universal/utils/fromNow.js

```diff
@@ -0,0 +1,28 @@
+const thresholds = {
```

Comment:
Heh, tryna get me on that twidar huh? I think I'll turn it into an npm package then, since this is completely void of application-specific logic.